packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantAvailability/_tests/RestaurantAvailability.test.js

```diff
@@ -0,0 +1,96 @@
+import { mount } from '@vue/test-utils';
+import { ClockSmallIcon, WalkingSmallIcon } from '@justeat/f-vue-icons';
+import availabilityTypes from '../availabilityTypes';
+import sut from '../RestaurantAvailability.vue';
```

Comment:
is it a widely used term? I wonder if just `component` or something like this will be more descriptive?

Reply:
It stands for `system under test`. It's a bit of an old habit from previous projects when writing test code. I believe it's relatively well known but perhaps not so much in the world of front-end? I am more than happy to rename it to something more widely understood

Reply:
>is it a widely used term? I wonder if just component or something like this will be more descriptive?
SUT did always confuse me too and lowecase sut feels more like a name that an acronym - maybe, for readability, we should create a ticket to rename all to component (or something similar)

Reply:
happy to do that! will fix forward :)